dev-requirements.txt

```diff
@@ -34,4 +34,4 @@ mypy==0.782
 # DEPENDENCY CHECKING #
 #######################
 
-safety~=1.9.0
+safety==1.9.0
```

Comment:
Don't freeze safety. We want this to be as new as possible without having to manually update it via automated PRs